src/Core/Addon/Module/ModuleManager.php

```diff
@@ -224,6 +224,48 @@ public function enable($name)
     }
 
     /**
+     * Disable a module specifiquely on mobile.
+     *
+     * @param  string $name The module name to disable
+     * @return bool         True for success
+     */
+    public function disable_mobile($name)
```

Comment:
camelCase notation please (for consistency with `isEnabled`), same for `enable_mobile`